src/runner.js

```diff
@@ -12,6 +12,10 @@ export function run(stdout, stdin, argv) {
       'only the rules specified will be used to validate the schema. Example: FieldsHaveDescriptions,TypesHaveDescriptions'
     )
     .option(
+      '-c, --config-directory <path>',
+      'path to begin searching for config files.'
+    )
```

Comment:
I completely agree with you that this will be 99% of the use cases.  I'm actually working on another project where we need to be able to supply the location of where the config files are, though, so I figured others might need this, too.  Are you cool with keeping?